components/server/src/prebuilds/bitbucket-server-service.ts

```diff
@@ -63,29 +63,8 @@ export class BitbucketServerService extends RepositoryService {
             console.log(`BBS: could not read webhooks.`, error, { error, cloneUrl });
             return false;
         }
-
-        if (repoKind === "users") {
-            const ownProfile = await this.api.getUserProfile(user, identity.authName);
-            if (owner === ownProfile.slug) {
-                return true;
-            }
-        }
-
-        let permission = await this.api.getPermission(user, { username: identity.authName, repoKind, owner, repoName });
-        if (!permission && repoKind === "projects") {
-            permission = await this.api.getPermission(user, { username: identity.authName, repoKind, owner });
-        }
-
-        if (this.hasPermissionToCreateWebhooks(permission)) {
-            return true;
-        }
-
-        console.log(`BBS: Not allowed to install webhooks.`, { permission });
-        return false;
-    }
-
-    protected hasPermissionToCreateWebhooks(permission: string | undefined) {
-        return permission && ["REPO_ADMIN", "PROJECT_ADMIN"].indexOf(permission) !== -1;
+        // return true once it can get webhooks, fallback to let SCM itself to check permission
```

Comment:
where fallback happens? 

It seems the code before that we have create permissions, not only read. Is it because it is not so fine grained in bbs?

Reply:
@akosyakov  We will add webhooks when creating the project. So BBS will go check it as test result shows https://github.com/gitpod-io/gitpod/pull/18575#issuecomment-1688163015

Reply:
Fallback because our permission checking for webhook add is not align to what BBS does
- Admin of BBS has all permission
- Admin can create webhook for all repo/project
- Admin don't have project_admin and repo_admin permission in its open APIs if they don't configure it manually in BBS projects/repos